.github/workflows/scorecards.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
     steps:
       - name: "Checkout code"
-        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846 # v3.0.0
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3.0.0
```

Comment:
Latest version id is 3.0.2

https://github.com/actions/checkout/blob/main/CHANGELOG.md